manon/components/checkbox.scss

```diff
@@ -47,41 +31,37 @@
       }
     }
   }
+}
 
-  &.required {
-    flex-direction: column;
-    align-items: flex-start;
+@mixin checkbox {
+  .checkbox {
+    padding: 0;
+    margin: 0 0 0.5rem;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    gap: theme.$checkbox-gap;
+    align-items: theme.$checkbox-align-items;
 
-    @include theming.apply-from-theme(
-      (
-        gap: theme.$checkbox-required-gap,
-        justify-content: theme.$radio-align-items,
-      )
-    );
+    @include checkbox-input;
```

Comment:
If it's only used once I wouldn't use a mixin. As it adds a layer of complexity.